less/mixins.less

```diff
@@ -1 +1,28 @@
 // Mixins below variables
+.button-creator(@background-color, @text-color, @border-color) {
+    display: inline-block;
+    text-decoration: none;
+    text-align: center;
+    font-size: 1.8rem;
+    color: @text-color;
+    padding: 11px 15px;
+    background: @background-color;
+    border: 1px solid @border-color;
+    border-radius: 5px;
+
+    @media (@phone) {
+        font-size: 1.6rem;
+        padding: 16px 0;
+    }
+}
+
+.flex-space-between {
+    display: flex;
+    justify-content: space-between;
+}
```

Comment:
So this is good. Even though you're creating a mixin for two styles, I 100% feel that is okay because you will be placing it in multiple different places. So good job using them.